### ISSUE TITLE
How to enable warmup epoch in finetuning spunet to reproduce result of MSC?

### ISSUE THREAD

**Issue by LydJason** [Pointcept/Pointcept#391]
Thank you for the great work!
When finetuning the spunet with pretrained MSC, I noticed that `warmup_epoch` was used, as shown in the appendix of MSC in the CVPR23 paper:
![image](https://github.com/user-attachments/assets/2d784b39-9f4f-4b20-9339-654146494154)
Sadly, I haven't found where to set this hyperparameter (Only found RuntimeProfiler, not sure if it is right to use), could you please show me how to set it? Is it critical to the result? Thank you!

**Comment by Gofinge** [comment]
Here (https://github.com/Pointcept/Pointcept/blob/main/configs/scannet/pretrain-msc-v1m1-0-spunet-base.py#L44)
https://pytorch.org/docs/stable/generated/torch.optim.lr_scheduler.OneCycleLR.html
Usually we set the `pct_start` number to 0.05
